docs/data/date-pickers/custom-components/custom-components.md

```diff
@@ -251,6 +251,13 @@ You can pass custom components to replace the icons, as shown below:
 
 {{"demo": "ArrowSwitcherComponent.js", "defaultCodeOpen": false}}
 
+## Access date adapter
+
+In case you are building a custom component that needs to work with multiple date libraries, you can access the date adapter instance by using the `usePickerAdapter` hook.
+This hook returns the date adapter instance used by the picker, which you can use to format dates, parse strings, and perform other date-related operations.
```

Comment:
Could we add a `:::success` saying that, if you are always working with the same date library, please use it directly to avoid having to migrate when we do BCs on the adapter?

Reply:
Great idea. 👍
Done. 😉